docs/stencil-docs/customizing-checkout/checkout-sdk-quickstart.md

```diff
@@ -37,171 +34,28 @@ For example, `cd cornerstone`
-
-`checkout: () => import('./theme/checkout'),`
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name">app.js</div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: "app.js"
-subtitle: ""
-lineNumbers: true
--->
-
-```js
-...
-const pageClasses = {
-    account_orderstatus: getAccount,
-    account_order: getAccount,
-    account_addressbook: getAccount,
-    shippingaddressform: getAccount,
-    account_new_return: getAccount,
-    'add-wishlist': () => import('./theme/wishlist'),
-    account_recentitems: getAccount,
-    account_downloaditem: getAccount,
-    editaccount: getAccount,
-    account_inbox: getAccount,
-    account_saved_return: getAccount,
-    account_returns: getAccount,
-    account_paymentmethods: getAccount,
-    account_addpaymentmethod: getAccount,
-    account_editpaymentmethod: getAccount,
-    login: getLogin,
-    createaccount_thanks: getLogin,
-    createaccount: getLogin,
-    getnewpassword: getLogin,
-    forgotpassword: getLogin,
-    blog: noop,
-    blog_post: noop,
-    brand: () => import('./theme/brand'),
-    brands: noop,
-    cart: () => import('./theme/cart'),
-    category: () => import('./theme/category'),
-		checkout: () => import('./theme/checkout'),
-    compare: () => import('./theme/compare'),
-    page_contact_form: () => import('./theme/contact-us'),
-    error: noop,
-    404: noop,
-    giftcertificates: () => import('./theme/gift-certificate'),
-    giftcertificates_balance: () => import('./theme/gift-certificate'),
-    giftcertificates_redeem: () => import('./theme/gift-certificate'),
-    default: noop,
-    page: noop,
-    product: () => import('./theme/product'),
-    amp_product_options: () => import('./theme/product'),
-    search: () => import('./theme/search'),
-    rss: noop,
-    sitemap: noop,
-    newsletter_subscribe: noop,
-    wishlist: () => import('./theme/wishlist'),
-    wishlists: () => import('./theme/wishlist'),
-};
-...
-```
-
-3. Save the file.
-
-## Preparing the <span class="fn">checkout.html</span> template file
-
-1. Open the <span class="fn">checkout.html</span> file in your text editor.
-2. Comment out the following statement:
-
- `{{{ checkout.checkout_content }}}`
-
-`<!--{{{ checkout.checkout_content }}}-->`
-
-3. On the next line, add the following:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name">checkout.html</div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: "checkout.html"
-subtitle: ""
-lineNumbers: true
--->
-
-```html
-<script>window.__webpack_public_path__ = "{{cdn 'assets/dist/'}}";</script>
-<script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
-
-<script>
-    window.stencilBootstrap("{{page_type}}", {{jsContext}}).load();
-</script>
-```
-
-4. Save the file.
+## Obtaining the JavaScript loader file

```

Comment:
I believe it would be better that for the "Checkout SDK quickstart" we do not reference Open Source Checkout. We could offer the following as a loader file:

```markdown
## Creating a checkout-loader.js file

1. In your text editor, create a file called checkout-loader.js with the following content:
```

```js 
var script = document.createElement('script');
script.onload = function () {
    checkoutKitLoader.load('checkout-sdk')
        .then(function (module) {
            var checkoutService = module.createCheckoutService();
            return checkoutService.loadCheckout();
        })
        .then(function (state) {
            console.log('Checkout SDK Quickstart', state.data.getCheckout());
            document.getElementById('checkout-app').innerHTML = 'Checkout ID: ' + state.data.getCheckout().id;
        });
};
script.src = 'https://checkout-sdk.bigcommerce.com/v1/loader.js';
document.head.appendChild(script);
```

```markdown
## Uploading your custom checkout
// ... Upload to webdav instructions ... maybe link to KB artcile

## Enable custom checkout
// Go to CP > Advanced Settings > Checkout, enable custom checkout and enter `webdav://checkout-loader.js`
```